README.md

```diff
@@ -8,17 +8,19 @@ WinRT APIs are defined in `*.winmd` format, and C#/WinRT includes tooling that g
 
 ## C#/WinRT Architecture
 
-### Component Authors
+The C#/WinRT runtime assembly, `WinRT.Runtime.dll`, is required by all C#/WinRT assemblies.  It provides an abstraction layer over the .NET runtime, supporting both .NET Standard 2.0 and .NET 5. The runtime assembly implements several features for all projected C#/WinRT types, including WinRT activation, marshaling logic, and [COM wrapper](https://docs.microsoft.com/dotnet/standard/native-interop/com-wrappers) lifetime management.
```

Comment:
Remove ".NET Standard 2.0" from support. we currently aren't shipping that anywhere.

Reply:
we're not shipping the .NET Standard 2.0 winrt.runtime.dll, but we're still building and testing it.  If we choose not to mention that, structure.md should be updated in a couple places.